esignet-service/README.md

```
@@ -33,196 +43,241 @@ All build targets generate TLS signing material first (`make keys`), because the
-```bash
-cd esignet-service
-docker build -f Dockerfile -t esignet:latest .
+## Client management API
+
+All endpoints require `Authorization: Bearer <token>` where the token carries the scope configured by `CLIENT_MGMT_REQUIRED_SCOPE` (default `client_mgmt`). The token is validated against the JWKS endpoint.
+
+Responses follow the envelope `{"response": {...}}` on success and `{"errors": [{"errorCode": "...", "errorMessage": "..."}]}` on failure.
+
+### Create OIDC client
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)